### DISCUSSION TITLE
How to run an executable file in alpine container?

### DISCUSSION THREAD

**Discussion by yehching0109** [pyinstaller/pyinstaller#5924] [Help]
Hi,
I had finish the python code > executable file through pyinstaller, and i test it from linux env and it works fine.

So i create a dockerfile and copy the executable file in it, but it couldn't run.
![image](https://user-images.githubusercontent.com/30792655/122163652-178ee380-cea8-11eb-9d4d-849f3a4142e9.png)

I go inside the container and use "./file", but  "sh: ./file: not found".
I check the executable file is in there.

Thanks

**Comment by bwoodsend** [comment]
To run on Alpine you need to build on Alpine - or at least a Linux distribution which uses musl instead of glibc.
